src/main/java/pl/arturzgodka/controllers/SecurityController.java

```diff
@@ -0,0 +1,62 @@
+package pl.arturzgodka.controllers;
+
+import jakarta.servlet.ServletException;
+import jakarta.servlet.http.HttpServletRequest;
+import org.springframework.security.core.Authentication;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestParam;
+import pl.arturzgodka.databaseutils.UserDao;
+import pl.arturzgodka.datamodel.*;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Controller
+public class SecurityController {
+
+    @GetMapping("/")
+    public String getLoginPage() {
+        return "index";
+    }
```

Comment:
Will this be removed here or in another PR?

Reply:
Also you can just reuse the webpage URL, this method is completely redundant

Reply:
I just removed MainController class which contained only this one routing, so after adding spring security and security controller it is no longer needed because my main page (index) is also my login page, so I keep it security controller.

Reply:
It will be removed here.